src/tree/SubscriptionTreeItem.ts

```diff
@@ -123,6 +124,7 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         await wizard.execute();
         context.showCreatingTreeItem(newStaticWebAppName);
 
+        await gitPull(nonNullProp(wizardContext, 'repo'));
         if (wizardContext.fsPath && hasRemote) {
             await updateWorkspaceSetting(appSubpathSetting, wizardContext.appLocation, wizardContext.fsPath);
```

Comment:
> I wanted to remove the settings from being added since with the current workspace requirements, it'll add the settings silently every time.

Does this modify settings.json even if the setting matches the default value? Ideally it does not

Reply:
Yeah, good call.

Reply:
There are enough changes here that I'll handle this in another PR.